File: 1008.java
import java.io.IOException;
import java.util.Scanner;
import java.text.DecimalFormat;

public class Main {
    public static void main(String[] args) throws IOException {
 
		Scanner input = new Scanner(System.in);
		DecimalFormat df = new DecimalFormat("0.00");
		int NUMBER, HOURS;
		double VALUE, SALARY;

		NUMBER = input.nextInt();
		HOURS = input.nextInt();
		VALUE = input.nextDouble();

		SALARY = HOURS * VALUE;
		System.out.println("NUMBER = " + NUMBER);
		System.out.println("SALARY = U$ " + df.format(SALARY));

    }
}

File: 1008.py
# -*- coding: utf-8 -*-

numero_funcionario = int(input())
horas_trabalhadas = int(input())
valor_hora = float(input())

salario = float(horas_trabalhadas * valor_hora) 

print("NUMBER = %d" %numero_funcionario)
print("SALARY = U$ %0.2f" %salario)
